src/App.js

```diff
@@ -28,5 +28,34 @@ Feel free to have fun and experiment with others too!
 If you get stuck or need help, please reach out on Slack.
 */
 export default function App() {
-  return <div>Replace this with your beautiful JSX</div>;
+  const members = team;
```

Comment:
You were the only one to experiment with JS variables before the return, nice job!